src/js/Autocompletes/Autocomplete.js

```diff
@@ -868,15 +869,15 @@ export default class Autocomplete extends PureComponent {
     }
   };
 
-  _setField = (field) => {
+  _setField(field) {
```

Comment:
Setting this back to an arrow function should fix the failed tests. Might've been an accidental change?